### PR TITLE
Renamed video files to match police-brutality titles

### PR DESCRIPTION
**Pull request by k-berryman**
My goal was to rename the .mp4 files from random names to names that match the description names in https://github.com/2020PB/police-brutality/tree/master/reports. This will make the videos be able to matched to their descriptions more quickly.

I also added an Alabama folder with files that were linked on https://github.com/2020PB/police-brutality/tree/master/reports but couldn't be found in the video folders.

Basically, I'm working towards including intention-revealing names and organizing the videos to match the descriptions.

### CODE REVIEW

**Review by wwwroth — APPROVED** [review]

**Comment by wwwroth** [comment]
This looks great- I was thinking of doing something similar. I saw a PR in the main repository to add unique IDs to each event and think they should be leveraged here as well.
